core/src/main/java/com/datastrato/gravitino/catalog/TableOperationDispatcher.java

```diff
@@ -190,20 +190,30 @@ public Table createTable(
                     .build())
             .build();
 
+    boolean opEntityFail = false;
     try {
       store.put(tableEntity, true /* overwrite */);
+    } catch (NoSuchEntityException noSuchEntityException) {
```

Comment:
Maybe we should try to import the non-existent schema first, not only add a warning log. WDYT? @jerryshao

Reply:
i think its a good idea。because there are too many schema not in the entity db in product environment

Reply:
We don't guarantee to write the entity successfully before. If we import the schema, we should reconsider the semantic. Does the `DropTable` and `AlterTable` need to import schema? Should we guarantee to write entity successfully if we consider importing schema?